main/migrations/0007_auto_20180603_0751.py

```diff
@@ -0,0 +1,20 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.10 on 2018-06-03 07:51
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('main', '0006_auto_20180603_0745'),
```

Comment:
why is there a dependencies on something that looks like a date file?

Reply:
This might be a database snapshot? That would make sense as a dependency for a migration. Is that right @Megh-Thakkar? 

Would be interesting to know what happens if this isn't available.

Reply:
Yes @DennisSchwartz , migrations files are a way Django keeps track of the database changes made. They work in a reverse manner. A file with the name '0007...' would be dependent on a file '0006...'. 

To understand this, consider the above situation. Before hand, the 'name' field had a length of 50 characters. I changed and made it 100(or 200). Now, this file stores this change. Since storing the existence of this field would be redundant, it creates a dependency on a previous file, a file which depends on the file before it(in this case, '0005...'), until it encounters the declaration of the 'name' field. Hence, the database stays updated!

Also, @DennisSchwartz, even if a single file in the migrations folder would get deleted, it will take pretty long to handle the error, since the database would not remain consistent. Hence, I have a backup on my system, github as well as a server that I own.

Reply:
Fantastic. Thanks Megh.